src/test/java/net/snowflake/client/core/arrow/DateConverterTest.java

```diff
@@ -162,4 +195,45 @@ public void testRandomDates() throws SFException {
       }
     }
   }
+
+  @Test
+  public void testTimezoneDates() throws SFException {
```

Comment:
have you run the tests also with JSON result format? is the result the same?

Reply:
Yep, the test actually has a line that compares the json converter with the arrow converter